xbmc/filesystem/CurlFile.h

```diff
@@ -209,5 +211,6 @@ namespace XFILE
       MAPHTTPHEADERS m_requestheaders;
 
       long            m_httpresponse;
+      bool            m_isshoutcast;
```

Comment:
Would you rather duplcate this **existing part** of CCurlFile
` if ((m_state->m_httpheader.GetProtoLine().substr(0, 3) == "ICY" || !m_state->m_httpheader.GetValue("icy-notice1").empty()
     || !m_state->m_httpheader.GetValue("icy-name").empty()
     || !m_state->m_httpheader.GetValue("icy-br").empty()) && !m_skipshout)
  {`
in CDVDFactoryInputStream::CreateInputStream?